velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -163,6 +163,14 @@ class StringTest : public SparkFunctionBaseTest {
         pos,
         len);
   }
+
+  std::optional<std::string> translate(
```

Comment:
Let's move it into 'translate' test as a lambda as well.

Reply:
Done. Thanks!